### PR TITLE
Remove config

### PR DESCRIPTION
**Pull request by Toby-Valtech**
# Description
This is an Improvement, where one Piece of Information/Configuration on what Software to Install was previously hidden in a Script. This was problematic because a Script is a bad place for configuration we want to change via Script, as breaking changes could easily be made. This has been fixed and the Configuration now resides in the Toolset.json for all Ubuntu Versions as discussed in [this](https://github.com/actions/virtual-environments/discussions/3535) discussion.
The new Version has been tested for Ubuntu-2004.
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/toolsets/toolset-1804.json [hunk 1]
```diff
@@ -68,6 +68,11 @@
             ]
         }
     ],
+    "tools":[
```
Comment (miketimofeev) [inline images/linux/toolsets/toolset-1804.json:71]: I don't think we need this section. We can take the tools from the `toolcache` block that have `"url"` property

File: images/linux/scripts/installers/Install-Toolset.ps1 [hunk 2]
```diff
@@ -52,7 +52,6 @@ foreach ($tool in $tools) {
         }
     }
 }
-
-chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/Python
-chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/node
-chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/go
\ No newline at end of file
+if ($toolsToInstall -contains "Python"){chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/Python}
```
Comment (miketimofeev) [inline images/linux/scripts/installers/Install-Toolset.ps1:55]: I think we can do foreach here:
```
foreach ($toolToInstall in  $toolsToInstall) {
    chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/$toolToInstall
}
```

File: images/linux/toolsets/toolset-1804.json (hunk 1 above)
Comment (maxim-lobanov) [reply images/linux/toolsets/toolset-1804.json:71]: Agree with Mikhail. We don't need the separate section.
We can just take the tools with `url` property. Or add separate flag property as an alternative.

**Comment by Toby-Valtech** [comment]
For legal reasons I will close this PR, and reopen it from my private account, as well as implementing the suggested changes.

**Pull request closed without merging**